client/src/main/java/org/asynchttpclient/DefaultAsyncHttpClient.java

```diff
@@ -33,6 +30,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.netty.channel.EventLoopGroup;
+import io.netty.util.HashedWheelTimer;
+import io.netty.util.Timer;
+
```

Comment:
...I guess my IDE is a little opinionated. Revert? Is there a style guide for this project, including a preferred import ordering?